### PR TITLE
RIA-9165 Bug fix, using absolute value for fee difference to avoid negatives

### PR DESCRIPTION
**Pull request by Ryan-Nesbitt**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/RIA-9165

### Change description ###

- Bug fix, using absolute value for fee difference ignoring negatives.

### Checklist

- [X] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by MiguelMartinRojas — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java [hunk 1]
```diff
@@ -291,7 +291,7 @@ public static String calculateFeeDifference(String originalFeeTotal, String newF
 
             BigDecimal originalFee = new BigDecimal(String.valueOf(Double.parseDouble(originalFeeTotal) / 100));
             BigDecimal newFee = new BigDecimal(String.valueOf(Double.parseDouble(newFeeTotal) / 100));
-            BigDecimal difference = originalFee.subtract(newFee);
+            BigDecimal difference = originalFee.subtract(newFee).abs();
```
Comment (MiguelMartinRojas) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java:294]: can you add a test to prevent negative numbers?

**Review by MiguelMartinRojas — CHANGES_REQUESTED** [review]

**Review by ikirsanov — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java (hunk 1 above)
Comment (Ryan-Nesbitt) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java:294]: Thanks Miguel I added a test case for 3 scenarios, positive difference, negative difference and no difference

**Review by MiguelMartinRojas — APPROVED** [review]
